src/components/dialog-full-screen/dialog-full-screen.stories.js

```diff
@@ -131,6 +154,41 @@ function makeButtonStory(name, themeSelector, stickyFooter, disableChromatic = f
   return [name, component, metadata];
 }
 
+function makeNested(name, themeSelector, disableChromatic = false) {
```

Comment:
I think we need to exclude `chromatic` here by providing `true` by `default`

Reply:
this is a default value, it is set to true on line 205 so we're fine here.
I will completely remove that argument as it is redundant.